beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/MySqlTestUtils.java

```diff
@@ -61,59 +62,59 @@
     connection = DriverManager.getConnection(jdbcUrl, username, password);
   }
 
-  void insertPath(String path, String table) throws SQLException {
+  void insertPath(String beekeeperTableName, String path, String table) throws SQLException {
```

Comment:
Why did this method change? Looks like its only used in one way

Reply:
Because I was getting some issues with the integration tests when I changed the table name of the `path` table to `housekeeping_path`. The `path` was hardcoded in before so o I changed the code to take in the table name as an option